cts/cli/regression.crm_mon.exp

```diff
@@ -23,7 +23,7 @@ Active Resources:
 =#=#=#= End test: Basic text output - OK (0) =#=#=#=
 * Passed: crm_mon        - Basic text output
 =#=#=#= Begin test: XML output =#=#=#=
-<pacemaker-result api-version="2.2" request="crm_mon --output-as=xml">
+<pacemaker-result api-version="2.3" request="crm_mon --output-as=xml">
```

Comment:
Not for this PR, but I'm thinking we should sed this to api-version="X" for comparison

Reply:
Done - https://github.com/ClusterLabs/pacemaker/compare/master...clumens:crm_mon-regression?expand=1